src/app/item-popup/ItemPopupBody.tsx

```diff
@@ -50,6 +52,27 @@ export default function ItemPopupBody({
     });
   }
 
+  const changeToTriageTab = useCallback((event: KeyboardEvent) => {
+    if (tabs.length < 2) {
+      return;
+    }
+    event.preventDefault();
+    onTabChanged(tabs[1].tab);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const changeToOverviewTab = useCallback((event: KeyboardEvent) => {
+    if (tabs.length < 2) {
+      return;
+    }
+    event.preventDefault();
+    onTabChanged(tabs[0].tab);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useHotkey('right', 'Switch to Triage tab', changeToTriageTab);
+  useHotkey('left', 'Switch to Overview tab', changeToOverviewTab);
```

Comment:
These strings need to be localized, see <https://github.com/DestinyItemManager/DIM/blob/master/docs/CONTRIBUTING.md#development>